ariadne_codegen/settings.py

```diff
@@ -278,16 +279,17 @@ def resolve_headers(headers: Dict) -> Dict:
 
 
 def get_header_value(value: str) -> str:
-    env_var_prefix = "$"
-    if value.startswith(env_var_prefix):
-        env_var_name = value.lstrip(env_var_prefix)
+    replacements = {}
+    for env_var_name in re.findall(r"\$([A-z][A-z0-9_]*)", value):
         var_value = os.environ.get(env_var_name)
```

Comment:
```suggestion
    for env_var_name in re.findall(r"\$([A-z][A-z0-9_]*|{[A-z][A-z0-9_]*})", value):
        var_value = os.environ.get(env_var_name.removeprefix("{").removesuffix("}"))
```
... to support `${VARIABLES}` if desired.